sw/host/opentitanlib/src/transport/cw310/mod.rs

```diff
@@ -84,6 +85,19 @@ impl CW310 {
         Ok(())
     }
 
+    // Initialize the values of the output pins on the board.
+    fn init_pin_values(&self) -> anyhow::Result<()> {
+        let device = self.device.borrow();
+        device.pin_set_state(Self::PIN_TRST, true)?;
+        device.pin_set_state(Self::PIN_SRST, true)?;
+        device.pin_set_state(Self::PIN_TAP_STRAP0, false)?;
+        device.pin_set_state(Self::PIN_TAP_STRAP1, true)?;
```

Comment:
Just a note: With the current pin mapping, this actually would enable JTAG for the RV_DM TAP and prevent bootstrap, if not for the lifecycle state we run in.

Until we fix the pin mapping on the CW310, SPI will not be available if these pins are not changed on-demand.

Reply:
yep that's correct, see the full discussion [here](https://github.com/lowRISC/opentitan/pull/12633#discussion_r871938097).

Reply:
#12722 is tracking this.